### PR TITLE
Fix params for isActual function

### PR DESCRIPTION
**Pull request by kimkha**
The function will call from here: https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/datasource.js#L1743

I don't know how to revert my commits in #139, so I create new one...

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch?

**Comment by slnode** [comment]
Can one of the admins verify this patch?

File: lib/migration.js [hunk 1]
```diff
@@ -66,10 +66,11 @@ function mixinMigration(MsSQL) {
     });
   };
 
-  MsSQL.prototype.isActual = function(cb) {
+  MsSQL.prototype.isActual = function(models, cb) {
```
Comment (ssh24) [inline lib/migration.js]: This is a breaking change to the api.

@kimkha Please consider changing the fix to this format the same as [mysql](https://github.com/strongloop/loopback-connector-mysql/blob/master/lib/migration.js#L124)

**Comment by ssh24** [comment]
@slnode ok to test

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by kimkha** [comment]
Thanks

**Pull request merged**